main/store/actions/index.js

```diff
@@ -926,6 +971,26 @@ module.exports = {
 
       return requests
     })
+  },
+  setBalanceMode(u, newMode) {
+    u('main', (main) => {
+      main.balanceFetchMode = newMode
+      return main
+    })
+  },
+  tokenVisiblity(u, chain, address, hidden) {
```

Comment:
there's a typo here but I think this function can be named a little more clearly as its not obvious what it does